pkg/operator/configobservation/ingresses/observe_ingresses.go

```diff
@@ -51,5 +53,11 @@ func ObserveIngressDomain(genericListers configobserver.Listers, recorder events
 		}
 	}
 
+	if reflect.DeepEqual(routingDomain, currentRoutingDomain) {
```

Comment:
`routingDomain` and `currentRoutingDomain` are just strings; why use `reflect.DeepEqual` rather than `==`?

Reply:
just to make sure when they change we are safe